### ISSUE TITLE
[feature] Add the color LightGreen (LG) to the color maps

### ISSUE THREAD

**Issue by Halfwalker** [formatc1702/WireViz#271]
For automotive wiring all the colors are represented, except for Light Green.  The standard colors are
![image](https://user-images.githubusercontent.com/1365031/152188222-2bc50b28-08f4-4750-b03e-c5ad7de42667.png)
It would be nice to be able to specify colors via that code, but certainly not necessary.

**Comment by kvid** [comment]
Yes, this has been discussed earlier as well, e.g. in https://github.com/formatc1702/WireViz/issues/135#issuecomment-671061938, and the on-going PR #251 probably includes new methods for specifying colors. You can take a preliminary look at that branch, but note that this PR is still unfinished, and might change a lot before merging into the `dev` branch.

With the current version of WireViz, you can always [specify colors by hexadecimal RGB values,](https://github.com/formatc1702/WireViz/blob/master/docs/syntax.md#colors) like #00FF00 or #008800. If you really need to use `LG` as a color, nobody stops you from adding this in your local `wv_colors.py`, but then be aware that your harnesses containing such custom colors cannot be processed correctly by normal WireViz installations (e.g. if you share such a harness YAML file with others).
